baseband/vlbi_base/base.py

```diff
@@ -167,6 +159,15 @@ def thread_ids(self):
     def thread_ids(self, ids):
         self._thread_ids = ids
 
+    @property
```

Comment:
I think for now I would do without the getter and setter, and just make this an attribute... Eventually, for proper VLBI classes we probably want to check `sample_rate` is consistent with an integer frame rate.